django-stubs/db/backends/mysql/base.pyi

```diff
@@ -39,20 +40,20 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     validation_class: type[DatabaseValidation]
 
     vendor: str
-    data_types: Any
-    operators: Any
+    data_types: dict[str, str]
```

Comment:
This is a cached property. Maybe we should reflect that? https://github.com/django/django/blob/9a27c76021f934201cccf12215514a3091325ec8/django/db/backends/mysql/base.py#L139

Reply:
Thank you for the good point :) I will fix that part